UnitystationLauncher/Assets/StationHub.metainfo.xml

```diff
@@ -46,9 +46,9 @@
 </screenshots>
 
 <releases>
-	<release version="927" date="2020-11-15">
+	<release version="928" date="2021-01-31">
 		<description>
-			<p>Bug fixes and lots of work towards making crossplatform distribution easier</p>
+			<p>Update Packages StationHub to last.</p>
```

Comment:
This isn't really grammatically correct, change it to: "Update Station Hub dependencies." or something similar.